.github/workflows/update-homebrew-release.yml

```diff
@@ -69,3 +69,68 @@ jobs:
           publish: true
           token: ${{ secrets.GH_BOT_TOKEN }}
           validate: false
+
+  update-homebrew-prerelease:
+    if: >-
+      github.repository_owner == 'LizardByte' &&
+      !github.event.release.draft && github.event.release.prerelease
```

Comment:
This probably isn't the right approach because this workflow will run 15 times... once for every Sunshine workflow job that publishes a release asset.

I have an idea though if the only thing that needs to change is the class name.